### PR TITLE
Issue #294 - Dismiss Highlight Bookmark

### PR DESCRIPTION
**Pull request by FrancescoSTL**
@chrismore r?

### CODE REVIEW

**Comment by chrismore** [comment]
Tested this out locally and it works. r+ from a functional perspective.

**Comment by FrancescoSTL** [comment]
Thanks, @chrismore! @schalkneethling r?

File: index.js [hunk 1]
```diff
@@ -125,6 +125,9 @@ var LightweightThemeManager = Cu.import('resource://gre/modules/LightweightTheme
 var activeWindow = windowUtils.getMostRecentBrowserWindow();
 // the awesomebar node from the browser window
 var awesomeBar = activeWindow.document.getElementById('urlbar');
+// the bookmark button node from the browser window
+var bookmarkButton = activeWindow.document.getElementById('bookmarks-menu-button');
+for (var item in activeWindow)
```
Comment (schalkneethling) [inline index.js]: stray `for` loop?

**Comment by schalkneethling** [comment]
One seemingly stray `for` loop, other that that it looks good. Needs a rebase. r+ w/c

**Comment by FrancescoSTL** [comment]
Done. Merging now!

**Pull request merged**